test/tests/reporters/restartable_data_reporter/restartable_data_reporter.i

```diff
@@ -0,0 +1,26 @@
+[Mesh/gmg]
+  type = GeneratedMeshGenerator
+  dim = 1
+  nx = 1
+[]
+
+[Problem]
+  solve = false
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 2
+[]
+
+[Reporters/data]
+  type = RestartableDataReporter
+  include = 'FEProblemBase/*/time*'
```

Comment:
I think extending this to a case where you use a vector of includes/excludes might be beneficial to show users that hey this is how you separate the vector in this case. Even though the documentation states that `*` and `?` can be used but it gave me a regexy feeling where `,;` have meaning. Also, maybe emphasizing a little more in the documentation that only these are supported could be nice.

Reply:
> Also, maybe emphasizing a little more in the documentation that only these are supported could be nice.

How is this not explicit enough? It describes the options

```
The [!param](/Reporters/RestartableDataReporter/include) and [!param](/Reporters/RestartableDataReporter/exclude)
parameters enable the filtering of data by name. With these parameters, you can use `*` as a
wildcard match of any number of characters and `?` as a wildcard match for a single character.
```

I would really prefer not to make the test more complicated for the sake of having an extra line in the documentation include. The test requirements are more important to me than the documentation for an advanced object

Reply:
Okay, I understand your reasoning. I was just looking at this from a naive user's perspective. I am fine with this though, will resolve this.

Reply:
Thanks. I think it's a valid concern - but I don't wanna make the test test more than it needs to to satisfy the requirement.

I would be more concerned if this wasn't considered a more "advanced" object